### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by yama-jum** [chibivue-land/vueyous#2] [General]
- [x] make concept
- [ ] make issues/issue-template
- [ ] make milestones
- [x] book policy
- [ ] versioning flow & changelog
- [ ] decide which VueYous composables to make
- [x] customize vitepress
- [x] book page path
- [ ] architecture
- [x] contribution guide
- [x] deploy vitepress book to github pages
- [x] i18n/l10n
- [ ] chapter structure
- [ ] testing composables
- [ ] add demo ui to composables
- [ ] github actions
- [x] change VitePress theme
- [ ] install renovate
- [ ] markup lint
- [x] make logo for VueYous
- [ ] og images

**Comment by ubugeeei** [comment]
> make logo for VueYous

relates: #21

**Comment by pei-pay** [comment]
> book page path

relates: #17

**Comment by pei-pay** [comment]
> book policy

> make concept

relates: #16

**Comment by naokihaba** [comment]
>  contribution guide

relates: #40

**Comment by naokihaba** [comment]
This will be integrated into https://github.com/chibivue-land/vueyous/issues/44.


I will also organize the Discussions when the time comes.
